### ISSUE TITLE
Mobs spawning stronger feature doesn't work very well.

### ISSUE THREAD

**Issue by xelotex** [Majrusz/MajruszsProgressiveDifficultyMod#237]
If you exit a world and re-enter the same world, mobs won't have the buffed stats anymore, but rather the vanilla stats.

Everything fine:
![2024-05-14_17 45 11](https://github.com/Majrusz/MajruszsProgressiveDifficultyMod/assets/70168682/9a4e19ed-e9cf-4b00-93a6-295873fdf106)

But when i re-enter the world, the extra stats are gone and the mob has vanilla stats:
![2024-05-14_17 48 57](https://github.com/Majrusz/MajruszsProgressiveDifficultyMod/assets/70168682/58d9a0e5-5bd0-4dd4-bd12-15bd9049028c)

**Comment by Chooksta** [comment]
Yeah I'm currently trying to find a solution for this as well. I've had to reset the values multiple times now due to them going back to their default values.
